src/main/resources/edu/cornell/kfs/fp/businessobject/datadictionary/PurchasingDataRecord.xml

```diff
@@ -0,0 +1,295 @@
+    <property name="label" value="Purchasing Data Id" />
+    <property name="forceUppercase" value="false" />
+    <property name="maxLength" value="9" />
+    <property name="control">
+      <bean parent="TextControlDefinition" p:size="9"/>
+    </property>
+  </bean>
+  
+  <bean id="PurchasingDataRecord-transactionSequenceRowNumber" parent="PurchasingDataRecord-transactionSequenceRowNumber-parentBean" />
+  <bean id="PurchasingDataRecord-transactionSequenceRowNumber-parentBean" abstract="true" parent="AttributeDefinition">
+    <property name="name" value="transactionSequenceRowNumber" />
+    <property name="forceUppercase" value="false" />
+    <property name="maxLength" value="9" />
+    <property name="control">
+      <bean parent="TextControlDefinition" p:size="9"/>
+    </property>
+  </bean>
+
+  <bean id="PurchasingDataRecord-accountNumber" parent="PurchasingDataRecord-accountNumber-parentBean" />
+  <bean id="PurchasingDataRecord-accountNumber-parentBean" abstract="true" parent="AttributeDefinition">
+    <property name="name" value="accountNumber" />
+    <property name="label" value="Account Number" />
+    <property name="forceUppercase" value="false" />
+    <property name="maxLength" value="16" />
+    <property name="control">
+      <bean parent="TextControlDefinition" p:size="15"/>
+    </property>
+  </bean>
+
+  <bean id="PurchasingDataRecord-tsysTranCode" parent="PurchasingDataRecord-tsysTranCode-parentBean" />
+  <bean id="PurchasingDataRecord-tsysTranCode-parentBean" abstract="true" parent="AttributeDefinition">
+    <property name="name" value="tsysTranCode" />
+    <property name="label" value="TSYS Tran Code" />
+    <property name="forceUppercase" value="false" />
+    <property name="maxLength" value="4" />
+    <property name="control">
+      <bean parent="TextControlDefinition" p:size="15"/>
+    </property>
+  </bean>
+
+  <bean id="PurchasingDataRecord-itemCommodityCode" parent="PurchasingDataRecord-itemCommodityCode-parentBean" />
+  <bean id="PurchasingDataRecord-itemCommodityCode-parentBean" abstract="true" parent="AttributeDefinition">
+    <property name="name" value="itemCommodityCode" />
+    <property name="label" value="Item Commodity Code" />
+    <property name="forceUppercase" value="false" />
+    <property name="maxLength" value="15" />
+    <property name="control">
+      <bean parent="TextControlDefinition" p:size="15"/>
+    </property>
+  </bean>
+
+  <bean id="PurchasingDataRecord-merchantOrderNumber" parent="PurchasingDataRecord-merchantOrderNumber-parentBean" />
+  <bean id="PurchasingDataRecord-merchantOrderNumber-parentBean" abstract="true" parent="AttributeDefinition">
+    <property name="name" value="merchantOrderNumber" />
+    <property name="label" value="Merchant Order Number" />
+    <property name="forceUppercase" value="false" />
+    <property name="maxLength" value="12" />
+    <property name="control">
+      <bean parent="TextControlDefinition" p:size="12"/>
```

Comment:
a common pattern I have seen in base code is that the control size for text controls is two larger than the maxLength for the field (so it would be 14 in this case); not sure if we've been following that pattern too, or if not, if we want to